src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -66,19 +70,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
   }
 
   /**
-   * Build and return an authorized API client service.
-   *
-   * @return an authorized API client service
-   * @throws GeneralSecurityException, IOException
+   * Applies parameters to the YouTube API, then extracts comments. URL is the only true variable;
+   * for this application we will always want order to be relevance and max results to be 100, the
+   * API's arbitrary limit
```

Comment:
Can we update this comment to reflect that it builds a request but does not actually send it to the Youtube API?

The current wording "Applies parameters to the Youtube API" is a little vague and seems to denote that it actually sends something externally.

Reply:
This does not seem resolved.